gnome-initial-setup/pages/network/gis-network-page.c

```diff
@@ -59,6 +59,16 @@ struct _GisNetworkPagePrivate {
 
   /* TRUE if the page has ever been shown to the user. */
   gboolean ever_shown;
+
+  /* If nm_device is in state ACTIVATED, but nm_client's connectivity is not
+   * yet FULL, the event source ID for a timeout. 0 otherwise, or after the
+   * timeout fires.
+   */
+  guint wait_connected_timeout_id;
```

Comment:
This needs to be removed and cleared in `dispose()` too, just in case the page somehow gets destroyed before it fires.

Reply:
Resolved in upstream version.